index.html

```diff
@@ -317,9 +317,8 @@ <h2>
           "powerful feature">name</dfn>, which is a string literal (e.g., "geolocation").
         </p>
         <p>
-          The user agent is responsible for tracking which <a>powerful features</a> each [=global
-          object/realm=] has the user's [=permission=] to use via the [=environment settings
-          object=].
+          The user agent tracks which <a>powerful features</a> the user has [=permission=] to use
+          via the [=environment settings object=].
```

Comment:
This implies that the user is forbidden from using some features, but it's really the user who grants permission for the environment settings objects (ESOs) to use the features.

I think this change is replacing Realm-scope with origin scope? So it might say:

```suggestion
          The user agent tracks which <a>powerful features</a> each [=origin=] has the user's [=permission=] to use....
```

I don't remember anymore why it's "via the ESO", and an origin tends to have multiple ESOs, so "the" isn't really the right article once things have transitioned to origins.

Reply:
right, I think we might need to reframe all this as part of #390 (as that will govern how permissions are associated with origins).